src/bin/mdbook-linkcheck.rs

```diff
@@ -47,6 +47,12 @@ struct Args {
         possible_values = &["always", "auto", "never"]
     )]
     colour: ColorChoice,
+    #[structopt(
+        short = "f",
+        long = "file",
```

Comment:
Not sure how the CLI will look for this, but maybe `long = "files"` instead?

```suggestion
        long = "files",
```